### DISCUSSION TITLE
What keypress visualisation overlay tool are you using in these demo gifs?

### DISCUSSION THREAD

**Discussion by airtonix** [akinomyoga/ble.sh#435] [General]
Curious, as screenkey is a bit meh.

![](https://github.com/akinomyoga/ble.sh/wiki/images/trial1.gif)

**Comment by akinomyoga** [comment]
I was asked that before (cf https://github.com/akinomyoga/ble.sh/issues/401#issuecomment-1925760672). It's [a modified version](https://github.com/akinomyoga/KeyCastOW) of [`brookhong/KeyCastOW`](https://github.com/brookhong/KeyCastOW), but it's a Windows software. I'm not sure if it works on Wine (or even on Wine/Wayland) because I haven't tried that.
